### PR TITLE
refactor: enable feature flag for Custom Views by default

### PR DESCRIPTION
**Pull request by emmenko**
We want to keep the flag as a long-lived flag (MC API), so that we can remove the one in LD.

### CODE REVIEW

File: packages/application-components/src/components/custom-views/custom-views-selector/use-custom-views-connector.ts [hunk 1]
```diff
@@ -23,8 +23,10 @@ type TUseCustomViewsFetcher = (props: TUseCustomViewsFetcherParams) => {
 export const useCustomViewsConnector: TUseCustomViewsFetcher = ({
   customViewLocatorCode,
 }) => {
+  const enableCustomViews = useFlagVariation(featureFlags.CUSTOM_VIEWS);
   const areCustomViewsEnabled =
-    useFeatureToggle(featureFlags.CUSTOM_VIEWS) &&
+    // @ts-ignore In case it's coming from the MC API, it's an object { value: boolean }.
+    (enableCustomViews?.value ?? enableCustomViews) &&
```
Comment (emmenko) [inline packages/application-components/src/components/custom-views/custom-views-selector/use-custom-views-connector.ts:29]: Just to evaluate it as a fallback.

File: packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ export const SetupFlopFlipProvider = (props: TSetupFlopFlipProviderProps) => {
 
   const defaultFlags = useMemo(
     () => ({
-      ...featureFlags.FLAGS,
+      ...featureFlags.DEFAULT_FLAGS,
```
Comment (emmenko) [inline packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx:155]: This is the main fix. We only want to pass default flags here, not the short-lived flags.

File: packages/constants/src/feature-toggles.ts [hunk 3]
```diff
@@ -21,6 +21,10 @@ export const CUSTOM_VIEWS = 'enableCustomViews';
...
 
 export const FLAGS = {
-  [CUSTOM_VIEWS]: false,
   [RECOLOURING]: false,
 };
+
+// Long-lived feature flags, defined in the MC API.
+export const DEFAULT_FLAGS = {
+  [CUSTOM_VIEWS]: { value: true },
+};
```
Comment (emmenko) [inline packages/constants/src/feature-toggles.ts:30]: Similarly to our internal setup, these are considered long-lived flags and they are defined as an object.

**Review by kark — APPROVED** [review]
💯
Thanks!

**Review by ddouglasz — APPROVED** [review]

**Pull request merged**